site/content/docs/5.3/examples/modals/index.html

```diff
@@ -129,10 +125,6 @@ <h1 class="fw-bold mb-0 fs-2">Sign up for free</h1>
           <small class="text-body-secondary">By clicking Sign up, you agree to the terms of use.</small>
           <hr class="my-4">
           <h2 class="fs-5 fw-bold mb-3">Or use a third-party</h2>
-          <button class="w-100 py-2 mb-2 btn btn-outline-secondary rounded-3" type="submit">
-            <svg class="bi me-1" width="16" height="16" aria-hidden="true"><use xlink:href="#twitter"/></svg>
-            Sign up with Twitter
-          </button>
```

Comment:
Maybe for purposes of this example, we use a random icon and a placeholder name? Could even swap for G Suite sign up tbh.